<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="output">
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="CmdInput" name="input"/>
         </item>
         <item>
          <widget class="QPushButton" name="historyBtn">
           <property name="text">
            <string>History..</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exceptionBtn">
           <property name="text">
            <string>Exceptions..</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QListWidget" name="historyList">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
     </widget>
     <widget class="QGroupBox" name="exceptionGroup">
      <property name="title">
       <string>Exception Handling</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>2</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="6">
        <widget class="QPushButton" name="clearExceptionBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Clear Stack</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="catchAllExceptionsBtn">
         <property name="text">
          <string>Show All Exceptions</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="catchNextExceptionBtn">
         <property name="text">
          <string>Show Next Exception</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="onlyUncaughtCheck">
         <property name="text">
          <string>Only Uncaught Exceptions</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="7">
        <widget class="QListWidget" name="exceptionStackList">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="7">
        <widget class="QCheckBox" name="runSelectedFrameCheck">
         <property name="text">
          <string>Run commands in selected stack frame</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="7">
        <widget class="QLabel" name="exceptionInfoLabel">
         <property name="text">
          <string>Stack Trace</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Filter (regex):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="filterText"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CmdInput</class>
   <extends>QLineEdit</extends>
   <header>.CmdInput</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
